src/gui/mzroll/mainwindow.cpp

```diff
@@ -2650,6 +2652,88 @@ void MainWindow::createMenus() {
     QAction* mlModelVideo = helpMenu->addAction("How ML model works");
     connect(mlModelVideo, &QAction::triggered, vidPlayer, &VideoPlayer::show);
 
+    QAction* sigabrtAction = helpMenu->addAction("Raise SIGABRT");
+    connect(sigabrtAction, &QAction::triggered, []() {
+        ::raise(SIGABRT);
+    });
+
+    QAction* sigfpeAction = helpMenu->addAction("Raise SIGFPE");
+    connect(sigfpeAction, &QAction::triggered, []() {
+        ::raise(SIGFPE);
+    });
+
+    QAction* sigillAction = helpMenu->addAction("Raise SIGILL");
+    connect(sigillAction, &QAction::triggered, []() {
+        ::raise(SIGILL);
```

Comment:
If this is just for testing purposes then should we not write tests for this instead ?
Feels a bit odd to have it  in the main UI

Reply:
Yes this is just temporary. I wanted to trigger exceptions and see how many of them were being captured on each platform. I left them here for the reviewer's test purposes as well. But this commit will be removed from the branch before it gets merged to develop.